protoc-artifacts/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM centos:6.6
+
+RUN yum install -y git \
+                   tar \
+                   wget \
+                   make \
+                   autoconf \
+                   curl-devel \
+                   unzip \
+                   automake \
+                   libtool \
+                   glibc-static.i686 \
+                   glibc-devel \
+                   glibc-devel.i686
+
+# Install Java 8
+RUN wget -q --no-cookies --no-check-certificate \
+    --header "Cookie: gpw_e24=http%3A%2F%2Fwww.oracle.com%2F; oraclelicense=accept-securebackup-cookie" "http://download.oracle.com/otn-pub/java/jdk/8u45-b14/jdk-8u45-linux-x64.tar.gz" \
+    -O jdk-8u45-linux-x64.tar.gz
```

Comment:
Not a big deal, but if you extract in the same line the download won't need to be stored in the docker image.

See https://github.com/grpc/grpc/blob/master/tools/dockerfile/grpc_java_base/Dockerfile#L46 . Note that if you "wget; tar ; rm file" it also wouldn't be saved in the image (if it is created and removed in the same RUN command).